e2e/tests/functional/search.e2e.spec.js

```diff
@@ -198,6 +198,32 @@ test.describe('Grand Search', () => {
     await expect(searchResultDropDown).toContainText('Clock A');
   });
 
+  test('Slowly typing after search debounce will abort requests @couchdb', async ({ page }) => {
+    let requestWasAborted = false;
+    //await createObjectsForSearch(page);
```

Comment:
remove dead code?

Reply:
agh, not dead code - should be in there to not depend on the previous test running in couch. i've readded - nice catch!